test/BUILD

```diff
@@ -412,11 +412,6 @@ apple_shell_test(
         "--ios_multi_cpus=i386,x86_64",
```

Comment:
@keith 
Should we drop support for `i386` cpu here?

Reply:
Yes it's fine to drop it.